### PR TITLE
Feat/result page

### PR DESCRIPTION
**Pull request by Aditi0505**
# Change Logs:

## Description:

- Added Result Page.
- Added all questions with user answer and correct answer.
- Fixed small css issues.

## Screenshot 
image
image

## Files to be ignored while reviewing
- None

## Preview Link:
https://deploy-preview-5--guessthisquiz.netlify.app/

### CODE REVIEW

**Review by pruthvirajmv — COMMENTED** [review]
Hi Aditi,
Liked the data modelling and keeping code clean
Please take care of minor flow fixes in the UI
Keep coding!

File: src/pages/Result/Result.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { Link } from "react-router-dom";
+import { ResultCard } from "../../components/Card/ResultCard";
+import { useCurrentQuestion } from "../../context";
+
```
Comment (govardhanreddyc79) [inline src/pages/Result/Result.jsx:4]: Can use absolute path here

**Comment by govardhanreddyc79** [comment]
Good folder structure.
Simple, neat UI.
Great work.

**Review by govardhanreddyc79 — APPROVED** [review]

**Review by anupkgurung — APPROVED** [review]
clean code.
good folder structure.

**Pull request merged**